client/device/cloudfmc/create.go

```diff
@@ -0,0 +1,121 @@
+package cloudfmc
+
+import (
+	"context"
+	"errors"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/device"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/device/application"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/goutil"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/http"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/retry"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/url"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/model/application/applicationstatus"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/model/devicetype"
+	"time"
+)
+
+type CreateInput struct {
+}
+
+func NewCreateInput() CreateInput {
+	return CreateInput{}
+}
+
+type createApplicationBody struct {
+	ApplicationType    string             `json:"applicationType"`
+	ApplicationStatus  string             `json:"applicationStatus"`
+	ApplicationContent applicationContent `json:"applicationContent"`
+}
+
+type applicationContent struct {
+	Type string `json:"@type"`
+}
+
+type CreateOutput = device.CreateOutput
+
+func Create(ctx context.Context, client http.Client, createInp CreateInput) (*CreateOutput, error) {
+
+	client.Logger.Println("creating cloud FMC")
```

Comment:
pls remove as it duplicated

Reply:
yes, done 👍

Reply:
Maybe relabel this to `Creating application object for cdFMC` (also cloud FMC is not what we call it; it's called cdFmc)

Reply:
done 👍